code/modules/clothing/modular_armor/attachments/storage.dm

```diff
@@ -179,6 +188,16 @@
 	icon_state = "mod_medic_bag"
 	storage =  /obj/item/storage/internal/modular/medical
 
+/obj/item/armor_module/storage/medical/freelancer
+
+/obj/item/armor_module/storage/medical/freelancer/Initialize()
```

Comment:
you dont need an empty type def, byond lets you do it just with redifing a new proc like initialize() here

Reply:
I know but this made me feel better about organization.

Reply:
it makes it look worse just remove it